src/test/data/JsonSerializableEndpointListTest/typicalEndpointList.json

```diff
@@ -1,32 +1,40 @@
 {
-  "_comment": "imPoster save file which contains the same Endpoint values as in TypicalEndpoints#getTypicalEndpointList()",
-  "endpoints" : [ {
-    "name" : "Alice Pauline",
-    "address" : "123, Jurong West Ave 6, #08-111",
-    "tagged" : [ "friends" ]
-  }, {
-    "name" : "Benson Meier",
-    "address" : "311, Clementi Ave 2, #02-25",
-    "tagged" : [ "owesMoney", "friends" ]
-  }, {
-    "name" : "Carl Kurz",
-    "address" : "wall street",
-    "tagged" : [ ]
-  }, {
-    "name" : "Daniel Meier",
-    "address" : "10th street",
-    "tagged" : [ "friends" ]
-  }, {
-    "name" : "Elle Meyer",
-    "address" : "michegan ave",
-    "tagged" : [ ]
-  }, {
-    "name" : "Fiona Kunz",
-    "address" : "little tokyo",
-    "tagged" : [ ]
-  }, {
-    "name" : "George Best",
-    "address" : "4th street",
-    "tagged" : [ ]
-  } ]
+	"_comment": "imPoster save file which contains the same Endpoint values as in TypicalEndpoints#getTypicalEndpointList()",
+	"endpoints": [
+		{
+			"name": "GET Pauline",
+			"address": "123, Jurong West Ave 6, #08-111",
+			"tagged": ["friends"]
+		},
+		{
+			"name": "Benson Meier",
+			"address": "311, Clementi Ave 2, #02-25",
+			"tagged": ["owesMoney", "friends"]
+		},
+		{
+			"name": "Carl Kurz",
+			"address": "wall street",
+			"tagged": []
+		},
+		{
+			"name": "Daniel Meier",
+			"address": "10th street",
+			"tagged": ["friends"]
+		},
+		{
+			"name": "Elle Meyer",
+			"address": "michegan ave",
+			"tagged": []
+		},
+		{
+			"name": "Fiona Kunz",
+			"address": "little tokyo",
+			"tagged": []
+		},
+		{
+			"name": "George Best",
```

Comment:
Would these test cases not being change cause the tests to break down?

Reply:
no, i got rid of all the tests using typicalendpointlist cause idk what they are for